### PR TITLE
minor: Use Composer binary-only images instead of installer script

### PR DESCRIPTION
**Pull request by Wirone**
Leverage Docker's `COPY --from` to copy `composer` binary from pre-built image ([more info](https://blog.codito.dev/2022/11/composer-binary-only-docker-images/)).

### CODE REVIEW

File: docker/php-7.4/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ FROM php:7.4-cli-alpine3.16
 ARG DOCKER_USER_ID
 ARG DOCKER_GROUP_ID
 
+# https://blog.codito.dev/2022/11/composer-binary-only-docker-images/
+COPY --from=composer/composer:2-bin /composer /usr/local/bin/composer
```
Comment (Wirone) [inline docker/php-7.4/Dockerfile:7]: If you guys don't like self-promotion like this, we can link composer/docker#250 instead 😉. I just wanted to provide background.

File: docker/php-7.4/Dockerfile [hunk 2]
```diff
@@ -3,6 +3,9 @@ FROM php:7.4-cli-alpine3.16
 ARG DOCKER_USER_ID
 ARG DOCKER_GROUP_ID
 
+# https://blog.codito.dev/2022/11/composer-binary-only-docker-images/
```
Comment (keradus) [inline docker/php-7.4/Dockerfile:6]:
```suggestion
# https://blog.codito.dev/2022/11/composer-binary-only-docker-images/
# https://github.com/composer/docker/pull/250
```

File: docker/php-7.4/Dockerfile
Comment (keradus) [inline docker/php-7.4/Dockerfile:1]: @Wirone , please check what do we consider a feature

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/feature-or-bug.rst

Comment (Wirone) [reply docker/php-7.4/Dockerfile:1]: Normally I would use `chore`, but as far as I remember PRLintReloaded does not allow it 😉. I'll change to `minor`.

**Review by kubawerlos — APPROVED** [review]

**Comment by kubawerlos** [comment]
thank you @Wirone

**Pull request merged**